tests/test_client.py

```diff
@@ -385,3 +385,29 @@ def test_list_trades():
                         'message': 'Trades Retrieved Successfully', 'payload': [{}]},
                   status=200)
     assert "Trades" in client.list_trades()['message']
+
+@responses.activate
+def test_eth_deposits_with_starkKey_success():
+    responses.post(url=f'{BASE_URL}/sapi/v1/payment/stark/start/',
+                    json={'status': 'success',
+                        'message': 'Success! Awaiting Blockchain Confirmation',
+                        'payload': ''})
+    res = client.crypto_deposit_start('100000','0x27..','0x27..','0x67..','930',65707,)
+    assert 'status' in res
+    assert 'success' == res['status']
+    assert 'payload' in res
+
+@responses.activate
+def test_eth_deposits_with_starkKey_fail():
+    responses.post(url=f'{BASE_URL}/main/payment/stark/start/',
+                    json={'status':'error',
+                        'message':'Essential parameters are missing',
+                        'payload': ''})
+    with pytest.raises(requests.exceptions.RequestException):
+        res = client.crypto_deposit_start('100000','0x27..','0x27..','0x67..','930',65707,)
+    # Handle exception
+        
+        data = res.response.json()
+        assert 'status' in data
+        assert data['status'] == 'error'
+        assert 'Essential parameters' in data['message']
```

Comment:
Can you research and add anymore edgecases?
Such as gas limit exceeded, not enough balance, etc